docs/build/html/_sources/quickstart.rst.txt

```diff
@@ -84,23 +84,26 @@ GoogleDrive treats everything as a file and assigns different mimetypes for diff
       return newFolder
 
 
-Get Id of the title
--------------------
+Return File ID via File Title
+-----------------------------
 
-``get_id_of_title`` is a function that return id of the given title's file in the directory.
+A common task is providing the Google Drive API with a file id.
+``get_id_of_title`` is a function that returns the id of a file handle by searching the file titles in a
+given directory. The function takes two arguments, title and parent_directory_id. Title is a string that
```

Comment:
I know that it's not a rule yet across these docs (they haven't been taken care of for a while at Google), but let's do `inline code` for var names - parent_directory_id, title, etc

Reply:
I updated the formatting. Will try and get the tests to run locally to see where this PR is having problems.

Reply:
ah, no need I think, CI fails because it can't run tests from a fork (tests depend on credentials that are needed to access GDrive test instance). I don't any reasons for this PR to break tests.

Reply:
I am also concerned with the missing lines of code. It appears this PR removes ~1,000 lines and I am not sure what they are for. If you want to review the updates and it looks good, go ahead and merge it. I may submit some changes in the future as I have started using this library for a project and want the docs to be clear for new team members and anyone else who starts using PyDrive.

Reply:
feel free to remove any changes to the `build` directory (just test it locally, that it builds w/o any warnings and you can open it). I can update the build later. (Actually, we need automate this process I think and this using GitHub actions + put build files to a different branch- feel free to create a ticket for this.)

>  I may submit some changes in the future as I have started using this library for a project and want the docs to be clear for new team members and anyone else who starts using PyDrive.

that would be great! thank you!